example/screens/ConnectWithRoomManagerScreen.tsx

```diff
@@ -0,0 +1,152 @@
+import React, {useState} from 'react';
+import {
+  Dimensions,
+  Image,
+  SafeAreaView,
+  StyleSheet,
+  Text,
+  View,
+} from 'react-native';
+import {connectScreenLabels} from '../types/ComponentLabels';
+import {useJellyfishClient} from '@jellyfish-dev/react-native-client-sdk';
+import {Button, TextInput, DismissKeyboard} from '../components';
+import {NativeStackScreenProps} from '@react-navigation/native-stack';
+import {AppRootStackParamList} from '../navigators/AppNavigator';
+import type {BottomTabScreenProps} from '@react-navigation/bottom-tabs';
+import {TabParamList} from '../navigators/AppNavigator';
+import {CompositeScreenProps} from '@react-navigation/native';
+import {usePermissionCheck} from '../hooks/usePermissionCheck';
+
+type Props = CompositeScreenProps<
+  BottomTabScreenProps<TabParamList, 'ConnectWithRoomManager'>,
+  NativeStackScreenProps<AppRootStackParamList>
+>;
+
+const {URL_INPUT, TOKEN_INPUT, CONNECT_BUTTON} = connectScreenLabels;
+
+async function getJellyFishServer(
+  roomManagerUrl: string,
+  roomName: string,
+  userName: string,
+) {
+  const url = roomManagerUrl.endsWith('/')
+    ? roomManagerUrl
+    : roomManagerUrl + '/';
+  const result = await fetch(
+    `${url}rooms/${roomName.trim()}/users/${userName.trim()}`,
```

Comment:
Consider adding
```suggestion
    `${url}api/rooms/${roomName.trim()}/users/${userName.trim()}`,
```
I'd expect the user to provide just protocol + host + port

Reply:
After recent API changes it indeed makes sense